src/views/BoardView.ts

```diff
@@ -11,10 +11,29 @@ class BoardView {
         const board = document.createElement(`div`);
         board.id = 'board';
         this.squareViews.forEach(({ element }) => board.appendChild(element));
+        this.generateNotation(board);
 
         document.querySelector('#app')?.appendChild(board);
     }
 
+    private generateNotation(board: HTMLElement): void {
+        const sides = ['left', 'right', 'bottom', 'top'];
+        const letters = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'];
+
+        sides.forEach((side, sideIndex) => {
+            const notationBox = document.createElement('div');
+            notationBox.classList.add('notation', `${side}-notation`);
+
+            for (let i = 8; i >= 1; i--) {
+                const notationElement = document.createElement('div');
+                notationElement.classList.add('notation-element');
+                notationElement.innerText = sideIndex < 2 ? i.toString() : letters[8 - i];
```

Comment:
można wydzielić metodę żeby nie zastanawiać się co to robi, typu assignLetterOrNumber. lub wydzielić z tego 2 metody oddzielną dla liter, oddzielna dla cyfr. tzn metoda będzie jedna tylko wywołana z innymi parametrami pewnie :)